jax/_src/scipy/linalg.py

```diff
@@ -46,17 +45,110 @@ def _cholesky(a: ArrayLike, lower: bool) -> Array:
   l = lax_linalg.cholesky(a if lower else jnp.conj(a.mT), symmetrize_input=False)
   return l if lower else jnp.conj(l.mT)
 
-@implements(scipy.linalg.cholesky,
-        lax_description=_no_overwrite_and_chkfinite_doc, skip_params=('overwrite_a', 'check_finite'))
+
 def cholesky(a: ArrayLike, lower: bool = False, overwrite_a: bool = False,
              check_finite: bool = True) -> Array:
+  """Compute the Cholesky decomposition of a matrix.
+
+  JAX implementation of :func:`scipy.linalg.cholesky`.
+
+  The Cholesky decomposition of a matrix `A` is:
```

Comment:
Nit: because `lower` is a thing, you might want to mention the upper decomposition as well.